app/_src/networking/meshservice.md

```diff
@@ -98,3 +98,47 @@ name of the `Dataplane` port.
     targetPort: 6739
     appProtocol: tcp
 ```
+
+## Migration
+
+MeshService is opt-in and involves a migration process. Every `Mesh` must enable
```

Comment:
todo: link to migraiton guide?

Reply:
This is the guide really. I added an explicit section with steps.